packages/url_launcher/url_launcher_web/example/pubspec.yaml

```diff
@@ -1,22 +1,22 @@
 name: regular_integration_tests
 publish_to: none
 
-environment:
-  sdk: ">=2.10.0-56.0.dev <3.0.0"
-
 dependencies:
   flutter:
     sdk: flutter
 
 dev_dependencies:
+  build_runner: ^1.10.0
+  mockito: ^5.0.0-nullsafety.5
+  url_launcher_web:
+    path: ../
   flutter_driver:
     sdk: flutter
   flutter_test:
     sdk: flutter
-  http: ^0.12.2
-  mockito: ^4.1.1
-  url_launcher_web:
-    path: ../
   integration_test:
-    path: ../../../integration_test
+    sdk: flutter
```

Comment:
Can you just opt out the integration test code (`// @dart=2.9`) for now as we've been doing in other packages, to avoid having plugins in the repo using different sources of `integration_test`? Then once this is on stable we can migrate everything.

Reply:
If I opt out the integration test code, this package has no tests :)

Reply:
> to avoid having plugins in the repo using different sources of `integration_test`?

What problems arise when different plugins are using difference sources of that package?

Reply:
> What problems arise when different plugins are using difference sources of that package?

I would expect none, since it is a dev_dependency, but dart deps still surprise me from time to time :/

My plan was to **not endorse** web plugins until all the standard plugins have rolled to integration_test from SDK, just to make sure that people can use them on the `stable` channel. 

(integration_test will not be cherry-picked for the upcoming stable, it'll probably come in the next one.)

Reply:
> > to avoid having plugins in the repo using different sources of `integration_test`?
> 
> What problems arise when different plugins are using difference sources of that package?

This comment wasn't prompted by a specific problem, just a preference for being able to easily reason about how `intgration_test` is working across this repo, with an all-at-once migration once it's fully ready.

Reply:
All right, here's the latest info about this. For now, `integration_test` from `sdk` is only used to run the integration tests of the `example` app of the package (with its own separate pubspec.yaml).

I've downgraded the SDK requirement of the *core* package, so it works in the upcoming `stable`.

The package will work in `stable` and up as normal (it'll even be endorsable). HOWEVER, the tests of the package (for now) will require `master` to run (or whatever channel contains the new null-safe integration_test package in the SDK).

Since Web tests need to be run manually (and they have never run automatically in flutter/plugins CI), I think this is an acceptable tradeoff. End users will have null-safe web support out of the box, and plugin developers will have a way to write and run tests for this package.

The "disparity" will only last until integration_test from SDK becomes widely available (which is going to be in the next push to Stable in April/May?), then even people in `stable` will be able to run the integration tests inside the `example` app.

How does this sound? @stuartmorgan

Reply:
I think that's probably fine. Realistically, I think the set of "people who need to run flutter/plugin integration tests" is probably entirely a subset of "people who are on master". If someone really wants to run integration tests on beta, they can revert those files locally. And if we find out that it's a more common use case than we think, we can revert them here.

(I'm still trying to figure out if there's a reasonable way for us to solve this problem for all the plugins, so that we're getting NNBD integration tests on master, but still running integration tests on beta and stable. Having no NNBD integration testing makes me pretty nervous.)